core/pkg/hostsensorutils/logging_messages.go

```diff
@@ -0,0 +1,10 @@
+package hostsensorutils
+
+// messages used for warnings
+var (
+	failedToGetData                     = "failed to get data"
```

Comment:
Please use the error convection.
e.g.
```
errFailedToGetData = errors.Error("failed to get data")
```

Reply:
Actually these are only warnings, not errors.
These are going to be used in `logger.L().Ctx(ctx).Warning(errFailedToGetData, helpers.Error(err))`, so we need to keep them as `string` since `Warning()` accept a `string` as argument.
Do you agree with me?